features/FEATURE_PSA/supporting_psa_in_mbed-os.md

```diff
@@ -40,8 +63,16 @@ Example:
     }
 ```
 
-### Adding single v7-M targets
-The PSA on single v7-M is supported with the help of Mbed OS PSA services. The following is taken from `targets/targets.json` and shows a PSA enabled target, `K64F`.
+## Naming convention for dual-core and Armv8-M target names
+As described in previous paragraphs, both SPE and NSPE target names **MUST** be defined for dual-core and Armv8-M targets. This section defines the naming convention for the same.
+
+`TargetA_S`       : PSA secure target (SPE)  
```

Comment:
What is "TargetA"? Does the "A" stand for something? Will folks adding boards think they need to put an "A" after their board name? Maybe use "TargetName" instead of "TargetA" to avoid such misunderstandings..

Reply:
Replaced `TargetA` with `TargetName`.